src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/SoundMonitor.java

```diff
@@ -0,0 +1,9 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers;
+
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+
+public class SoundMonitor {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)